dev/com.ibm.websphere.appserver.features/visibility/public/mcp-1.0/io.openliberty.mcp-1.0.feature

```diff
@@ -0,0 +1,23 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.mcp-1.0
+visibility=public
+IBM-App-ForceRestart: install, \
+ uninstall
+IBM-API-Package: \
+ io.openliberty.mcp, \
+ io.openliberty.mcp.annotations, \
+ io.openliberty.mcp.content, \
+ io.openliberty.mcp.encoders, \
+ io.openliberty.mcp.messaging, \
+ io.openliberty.mcp.meta, \
+ io.openliberty.mcp.tools
+IBM-ShortName: mcp-1.0
```

Comment:
`build.featureStart.base/resources/com/ibm/ws/test/featurestart/features/feature-levels.properties` will need to be updated to have `mcp-1.0=17` added to it.